### ISSUE TITLE
Complete freeze on MacOS Ventura 13.0.1

### ISSUE THREAD

**Issue by Bubobubobubobubo** [neovide/neovide#1805]
**Describe the bug**
Neovide works great until the graphic rendering suddenly freezes. After that, many possible paths:
- the graphic rendering comes back for a few frames after ~20s and stay frozen until the app is killed.
- the application freezes and then makes the whole system unresponsive at least for a brief moment (~10).

In either case, no error message is being printed or nothing more can help with the identification of the bug. Focusing the window becomes impossible. Processes currently running (such as Neovim term) and spawned by Neovim are still working and remain active. Killing the application also results in the OS freezing for 10s.

**To Reproduce**
1. Install the binary release of Neovide for MacOS.
2. Use the application, wait for the freeze to happen (can be 10s, can be 10min).

**Expected behavior**
The application should not freeze. If the graphical rendering freezes, killing the application would probably be preferable.

**Desktop (please complete the following information):**
 - OS: MacOS Ventura 13.0.1
 - Neovide Version: 0.10.3
 - Neovim Version: 0.8.3

**Please run `neovide --log` and paste the contents of the `.log` file created in the current directory here:**

Unavailable option for binary install directly in Applications folder.

**Additional context**

Can cursor animation be the culprit? I specifically installed Neovide to play with this feature. I am using the custom cursor FXs but no option is being pushed harder than it should be.

**Comment by declanchiu** [comment]
I also encountered this problem

**Comment by swithun-liu** [comment]
I try to open the app, but nothing happen, no app was opened

m1 mac
Ventura 13.0.1

**Comment by MicroBlueee** [comment]
I tried to open the app, but after opening it there is only a black screen and the program freeze. i can't open and edit any file either.
![截屏2023-03-24 16 46 08](https://user-images.githubusercontent.com/49147382/227469990-3ef2a3d5-e277-4e74-8fd8-a937f0e49fc2.png)
Intel Mac，Ventura 13.2.1

**Comment by Bubobubobubobubo** [comment]
@MicroBlueee, @georgeguimaraes : i don't think that we are referring to the same issue. My Neovide works correctly until it crashes all of the sudden. I never experienced any black screen.

**Comment by VidocqH** [comment]
Same on MacOS Ventura 13.0, works great until suddenly freeze.

No error or warning messages, nothing help.

The only info I can give is freeze usually happens after Neovide runs more than 10 hours, between 10-15 hours I suppose.

**Comment by pernambucano** [comment]
I'm encountering a similar problem, but in my case neovide eats 30 gb of ram after a few hours of use and then my mac freezes.

**Comment by VidocqH** [comment]
[neovidelog.txt](https://github.com/neovide/neovide/files/11122764/neovidelog.txt)

I encountered it again, here are logs display in crush window.

**Comment by BugLight** [comment]
> I try to open the app, but nothing happen, no app was opened
> 
> m1 mac Ventura 13.0.1

Hello everyone!
I managed to solve this problem locally.
I encountered the same problem on my M1 Mac Ventura 13.2.1 after I had installed neovide from homebrew.
Then I compiled neovide from source and that fixed it. Probably you should try building from source and see if it works for you.

**Comment by fredizzimo** [comment]
Closing this simce it's most likely already fixed.
